docs/src/architecture.md

```diff
@@ -128,7 +128,6 @@ router record for a `UAID`.
 | router_type  | Router Type (See [`autoendpoint::extractors::routers::RouterType`])              |
 | node_id      | Hostname of the connection node the client is connected to.                      |
 | connected_at | Precise time (in milliseconds) the client connected to the node.                 |
-| last_connect | **global secondary index** - year-month-hour that the client has last connected. |
```

Comment:
Since this is already legacy, I wonder if we should keep it? 

The page already discusses things like `curmonth`, which is also a DDB thing back when we did table rotations, so keeping the `last_connect` is about as equally relevant and historic. 

If we just wanted to clean things up, we'd drop all the legacy DDB documentation as well and just keep stuff that reflects the current state.